algebird-test/src/test/scala/com/twitter/algebird/BloomFilterTest.scala

```diff
@@ -51,6 +54,27 @@ class BloomFilterLaws extends CheckProperties {
     }
   }
 
+  property("the distance between a filter and itself should be 0") {
+    forAll { (a: BF[String]) =>
+      BloomFilter.hammingDistance(a, a) == 0
+    }
+  }
```

Comment:
can we do: `forAll { (a: BF[String]) => BloomFilter.hammingDistance(a, BFZero) == a.numBits }`?